src/app.ts

```diff
@@ -1,50 +1,55 @@
-import "reflect-metadata"; // this shim is required
-import { createExpressServer, useContainer } from "routing-controllers";
-import { Container } from "typedi";
-import { createConnection } from "typeorm";
+import 'reflect-metadata'; // this shim is required
+import { createExpressServer, useContainer } from 'routing-controllers';
+import { Container } from 'typedi';
+import { createConnection } from 'typeorm';
 import AdminBro from 'admin-bro';
 //TODO: Check how to change this to TS import
-const AdminBroExpress = require('@admin-bro/express')
+const AdminBroExpress = require('@admin-bro/express');
```

Comment:
I think for handle this type of imports you will need to configure `tsconfig.json` and  modify the `baseUrl` attribute to `.` as well as the other paths according to this path, in this way you can include `admin-bro`

Reply:
Thanks Nico, will take a look.